createdisk.sh

```diff
@@ -338,6 +341,9 @@ kernel_cmd_line=$(${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'cat /proc/cm
 # SCP the vmlinuz/initramfs from VM to Host in provided folder.
 ${SCP} core@api.${CRC_VM_NAME}.${BASE_DOMAIN}:/boot/ostree/rhcos-${ostree_hash}/* $1
 
+# Get the IP of the VM
+internalIP=$(${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'hostname -i')
```

Comment:
This returns multiple IP addresses on my crc VM and my laptop, how do we pick the right one? The manpage also recommends `-I` rather than `-i` If the etcd certificate or port is easy to access, we could get the IP to use directly from it.

Reply:
Alright I will use something like `internalIP=$(${DIG} +short api.${CRC_VM_NAME}.${BASE_DOMAIN})` then.